packages/forms/resources/css/components/markdown-editor.css

```diff
@@ -0,0 +1,31 @@
+@import 'easymde/dist/easymde.min.css';
+
+.EasyMDEContainer .editor-toolbar {
+    @apply hidden;
+}
+
+.EasyMDEContainer .CodeMirror {
+    @apply w-full h-full min-h-full transition duration-75 rounded-lg shadow-sm border border-gray-300 focus:border-primary-500 focus:ring-1 focus:ring-inset focus:ring-primary-600 caret-black !important;
+}
+
+.EasyMDEContainer .CodeMirror .CodeMirror-scroll {
+    @apply tracking-normal whitespace-pre-wrap font-mono text-sm !important;
```

Comment:
!important does not work with @apply in Sass, so these need to be normal CSS declarations :(

Reply:
Ah, that sucks. Important probably isn't even needed because of specificity.